### DISCUSSION TITLE
"How-to-enable-importing-and-exporting-different-file-formats-with-AbiWord" wiki page not public?

### DISCUSSION THREAD

**Discussion by marlop352** [ether/etherpad-lite#5817] [Q&A]
Is there any reason why the https://github.com/ether/etherpad-lite/wiki/How-to-enable-importing-and-exporting-different-file-formats-with-AbiWord page is not public?
Also is there any difference on the pdf exports of abiword and libreoffice?

**Comment by marlop352** [comment]
I'm following this documentation https://github.com/ether/etherpad-lite/blob/v1.9.1/doc/docker.adoc and abiword seems to not work while libreoffice does work, what logs and information is needed to debug this?

**Comment by SamTV12345** [comment]
Which docker container were you using? Could you please also try the develop tag. Maybe that will work.

**Reply by marlop352** [reply]
I was using the etherpad/etherpad:latest docker image, I gave up on making etherpad work for me +/- half a month ago so I don't have the environment to test stuff related to it anymore.

**Reply by SamTV12345** [reply]
Alright. Thanks for reporting back.
